src/Display.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Display from './Display';
+import { render } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+it('renders successfully', () => {
```

Comment:
good job. Usually, you'd have a description here for all the tests but since this test is in the display file it's a bit intuitive what this is for I suppose.